pkgs/development/libraries/tbb/default.nix

```diff
@@ -1,88 +1,28 @@
 { lib
 , stdenv
-, fetchurl
 , fetchFromGitHub
+, cmake
+, pkg-config
 , fixDarwinDylibNames
 }:
 
 stdenv.mkDerivation rec {
   pname = "tbb";
-  version = "2020.3";
+  version = "2021.7.0";
 
   src = fetchFromGitHub {
     owner = "oneapi-src";
     repo = "oneTBB";
     rev = "v${version}";
-    sha256 = "prO2O5hd+Wz5iA0vfrqmyHFr0Ptzk64so5KpSpvuKmU=";
+    sha256 = "sha256-Lawhms0yq5p8BrQXMy6dPe29dpSlHdSntum+6bAkpyo=";
   };
 
-  patches = [
-    # Fixes build with Musl.
-    (fetchurl {
-      url = "https://github.com/openembedded/meta-openembedded/raw/39185eb1d1615e919e3ae14ae63b8ed7d3e5d83f/meta-oe/recipes-support/tbb/tbb/GLIBC-PREREQ-is-not-defined-on-musl.patch";
-      sha256 = "gUfXQ9OZQ82qD6brgauBCsKdjLvyHafMc18B+KxZoYs=";
-    })
-
-    # Fixes build with Musl.
-    (fetchurl {
-      url = "https://github.com/openembedded/meta-openembedded/raw/39185eb1d1615e919e3ae14ae63b8ed7d3e5d83f/meta-oe/recipes-support/tbb/tbb/0001-mallinfo-is-glibc-specific-API-mark-it-so.patch";
-      sha256 = "fhorfqO1hHKZ61uq+yTR7eQ8KYdyLwpM3K7WpwJpV74=";
-    })
-
-    # Fixes build with upcoming gcc-13:
-    #  https://github.com/oneapi-src/oneTBB/pull/833
-    (fetchurl {
-      name = "gcc-13.patch";
-      url = "https://github.com/oneapi-src/oneTBB/pull/833/commits/c18342ba667d1f33f5e9a773aa86b091a9694b97.patch";
-      sha256 = "ZUExE3nsW80Z5GPWZnDNuDiHHaD1EF7qNl/G5M+Wcxg=";
-    })
-
-    # Fixes build for aarch64-darwin
-    (fetchurl {
-      name = "aarch64-darwin.patch";
-      url = "https://github.com/oneapi-src/oneTBB/pull/258/commits/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch";
-      sha256 = "sha256-JXqrFPCb3q1vfxk752tQu7HhApCB4YH2LoVnGRwmspk=";
-    })
-  ];
-
-  nativeBuildInputs = lib.optionals stdenv.isDarwin [
+  nativeBuildInputs = [ cmake pkg-config ] ++ (lib.optionals stdenv.isDarwin [
     fixDarwinDylibNames
```

Comment:
```suggestion
  nativeBuildInputs = [ cmake pkg-config ]
    ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
```

Reply:
Always using `optionals` and multi-line dependencies makes it less error-prone and more `git diff` and `blame` friendly.